### PR TITLE
OSDOCS#12021: Known issue with OLM v1 private registries

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.17

Issue: [OSDOCS-12021](https://issues.redhat.com//browse/OSDOCS-12021)

Link to docs preview:
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/arch/catalogd.html
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/catalogs/creating-catalogs.html
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/catalogs/fbc.html
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/catalogs/rh-catalogs.html
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/ce/managing-ce.html
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/ce/managing-ce.html#olmv1-installing-an-operator_managing-ce
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/ce/upgrade-edges.html
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/index.html#olmv1-highlights_olmv1-about
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/catalogs/managing-catalogs.html#olmv1-creating-a-pull-secret-for-catalogs-secure-registry_managing-catalogs
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/catalogs/managing-catalogs.html#olmv1-red-hat-catalogs_managing-catalogs
* https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/catalogs/managing-catalogs.html#olmv1-adding-a-catalog-to-a-cluster_managing-catalogs

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mwerner2113** [comment]
Should definitely be an admonition [here](https://82304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/catalogs/managing-catalogs.html#olmv1-red-hat-catalogs_managing-catalogs) instead of in _Managing Catalogs_. 

Think it's very possible that folks go straight to a procedure either in the TOC or from google search and then miss the admonition. Can make more suggestions of where it should go later tonight or tomorrow morning.

**Review by everettraven — APPROVED** [review]

**Comment by michaelryanpeter** [comment]
@kuiwang02, @bandrade, or @jianzhangbjz PTAL

File: snippets/olmv1-known-issue-private-registries.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// * extensions/catalogs/fbc.adoc
+// * extensions/catalogs/managing-catalogs.adoc
+// * extensions/catalogs/rh-catalogs.adoc
+// * extensions/ce/managing-ce.adoc
+// * extensions/ce/upgrade-edges.adoc
+// * extensions/index.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+Currently, {olmv1-first} cannot authenticate private registries, such as the Red Hat-provided Operator catalogs. This is a known issue. As a result, the {olmv1} procedures that rely on having the Red Hat Operators catalog installed do not work. (link:https://issues.redhat.com/browse/OCPBUGS-36364[*OCPBUGS-36364*])
```
Comment (kuiwang02) [inline snippets/olmv1-known-issue-private-registries.adoc:15]: @michaelryanpeter just FYI: it is for clusterextension which does not support pull image from private registry.
for clustercatalog, we can create secret to pull image from private registry, even for registry.redhat.io.

Comment (michaelryanpeter) [reply snippets/olmv1-known-issue-private-registries.adoc:15]: @kuiwang02 I created a ticket to revise this note after GA with a better description: https://issues.redhat.com/browse/OSDOCS-12125

Thank you!

File: modules/olmv1-red-hat-catalogs.adoc [hunk 2]
```diff
@@ -13,7 +13,19 @@
 
 [IMPORTANT]
 ====
+
+* {empty}
```
Comment (jeana-redhat) [inline modules/olmv1-red-hat-catalogs.adoc:17]: neat!

**Review by jeana-redhat — COMMENTED** [review]
Globally: s/Red Hat/Red{nbsp}Hat

Otherwise LGTM!

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/olmv1-red-hat-catalogs.adoc (hunk 2 above)
Comment (michaelryanpeter) [reply modules/olmv1-red-hat-catalogs.adoc:17]: It's super cool, right!

**Comment by michaelryanpeter** [comment]
ack'ed by PX via Content Strategy

**Pull request merged**